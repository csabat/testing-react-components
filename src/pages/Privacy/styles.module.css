.privacyContainer {
  width: 60%;
  margin: auto;
  padding-top: 5rem;
  justify-content: center;
}

.privacyContainer > p {
  margin-bottom: 1rem;
}