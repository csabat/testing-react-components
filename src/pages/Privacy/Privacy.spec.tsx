import React from 'react';
import { render } from '@testing-library/react';

import Privacy from './Privacy';

describe('Privacy', () => {
  it('renders the page', () => {
    const { container } = render(<Privacy/>);

    expect(container).toMatchSnapshot();
  });
});