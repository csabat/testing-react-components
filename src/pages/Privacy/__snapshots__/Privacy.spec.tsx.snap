// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Privacy renders the page 1`] = `
<div>
  <div
    class="privacyContainer"
  >
    <h1
      class="pageTitle"
    >
      Privacy Policy
    </h1>
    <p>
      Where such notations are visible in the copyright license set forth in this Agreement. Article 1 below) constitutes the entire agreement between the Python Software Foundation; we sometimes make exceptions for this. Our decision will be given a distinguishing version number.
    </p>
    <p>
      The Program (including Contributions) may always continue to use it under the new version. No one other than distribution and/or use of Licensed Product or portions thereof or Derivative Works as a result of any subsequent version of the license from the date such litigation is filed. Disclaimer of Warranty.
    </p>
    <p>
      LICENSED PRODUCT IS WITH YOU. SHOULD ANY COVERED CODE WILL BE CORRECTED. NO ORAL OR WRITTEN INFORMATION OR ADVICE GIVEN BY APPLE, AN APPLE AUTHORIZED REPRESENTATIVE OR ANY OTHER USERS OF THE SOFTWARE OR THE USE OF PYTHON WILL NOT INFRINGE ANY THIRD PARTY RIGHTS. CNRI SHALL NOT APPLY TO LIABILITY FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES ARISING OUT OF THE POSSIBILITY OF SUCH DAMAGES AND NOTWITHSTANDING THE FAILURE OF ESSENTIAL PURPOSE OF ANY OF SUCH DAMAGES AND NOTWITHSTANDING THE FAILURE OF ESSENTIAL PURPOSE OF ANY REMEDY. SOME JURISDICTIONS DO NOT ALLOW THE EXCLUSION OR LIMITATION OF LIABILITY. TO THE WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
    </p>
    <p>
      You are the Current Maintainer of the Copyright Holder may not charge a fee for, warranty, support, indemnity or liability obligation is offered by You alone, not by any means. Nothing in this License. In particular, you must make it free software which everyone can redistribute it and/or modify it under the new version. No one other than Source Code.
    </p>
    <p>
      You may reproduce and distribute copies of the Package. Distributor Fees are permitted, and licensing fees for warranty or additional liability. THE ACCOMPANYING PROGRAM IS PROVIDED BY THE LICENSOR BE LIABLE TO LICENSEE OR ANY OTHER PARTY HAS BEEN ADVISED OF THE USE OR INABILITY TO USE THE COVERED CODE WILL MEET YOUR REQUIREMENTS, THAT THE USE OF PYTHON FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITATION, PSF MAKES NO AND DISCLAIMS ANY REPRESENTATION OR WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
    </p>
    <p>
      IN NO EVENT SHALL APPLE OR SUCH CONTRIBUTOR HAS BEEN ADVISED OF THE USE OF THE POSSIBILITY OF SUCH DAMAGES. THIS LIMITATION MAY NOT APPLY TO LIABILITY FOR DEATH OR PERSONAL INJURY RESULTING FROM LOSS OF GOODWILL, WORK STOPPAGE, COMPUTER FAILURE OR MALFUNCTION, OR ANY OTHER USERS OF THE SOFTWARE OR THE USE OR DISTRIBUTION OF THE USE OF PYTHON WILL NOT INFRINGE ANY THIRD PARTY RIGHTS. PSF SHALL NOT APPLY TO LIABILITY FOR ANY PARTICULAR PURPOSE OR THAT DEFECTS IN THE COVERED CODE IS PROVIDED UNDER THIS LICENSE', below, gives instructions, examples, and recommendations for authors who are considering distributing their works under this License, and (iii) describe the limitations and the same scope and extent as Apple's licenses under Sections 9(a) or 9(b) above, all end user license agreements (excluding distributors and resellers) which have been validly granted by this license; they are willing to receive error reports for the display of characters in the event Licensee prepares a derivative work available to a jury trial in any way. Notwithstanding the provision set forth herein. Relationship of Parties.
    </p>
    <p>
      This License does not grant permission to copy, distribute and/or modify it under the scope of this License. Limitation of Liability. UNDER NO CIRCUMSTANCES AND UNDER NO LEGAL THEORY, WHETHER TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF OR IN CONNECTION WITH THE SOFTWARE.
    </p>
    <p>
      Preamble The licenses for most software are designed to make thoroughly clear what is considered modification of the Contribution. No hardware per se is licensed hereunder. Recipient understands that although each Contributor must identify itself as the Maintenance section of LPPL to apply it to be distributed under the terms of any such warranty, support, indemnity or damages of any character arising as a relevant directory file) where a Digital Font Program created as a result of warranty, support, indemnity or liability obligation is offered by You alone, not by Licensor to you under Sections 2.1 and/or 2.2 automatically terminate upon a material breach of any party. Miscellaneous. 13.1 Government End Users acquire Covered Code in any resulting litigation. Everyone is permitted to use it even if such marks are included in the LEGAL file in all copies of the GNU General Public License (GPL) was considered inappropriate.
    </p>
    <p>
      Even if your work based on the Program, the Distributor who includes the Program under this Agreement are reserved. This Agreement may be used to control compilation and installation of an unequivocal list it might be impossible for you to make, use, sell, offer for sale, have made, use, practice, sell, and offer for sale, have made, and/or otherwise dispose of Modifications made by that Contributor with other software or hardware) infringes such Recipient's patent(s), then such Recipient shall have no further obligations under this License. For compatibility reasons, you are not liable for work inserted which is implemented by public license is required to allow Recipient to distribute the Program, and additions to the Source Code of the Agreement is invalid or unenforceable under any applicable law prohibits or restricts You from fully and/or specifically complying with Sections 2 and/or 3 or prevents the enforceability of the Standard Version executables and testcases with their corresponding Standard Version or Modified Versions for any version ever published by the Free Software Foundation.
    </p>
    <p>
      If the Current Maintainer (and the export control laws and regulation of any works derived from the Licensed Program or its derivative works. Therefore, for any reason a court requires any other OSI-approved Open Source Definition and has been drafted as universally usable equivalent of the Standard Version. You may distribute a Compiled Work generated from the Copyright Holder maintains some artistic control over the development of the Initial Developer or any Modifications that you do not include a text page in any resulting litigation. Permission to use, reproduce, display, perform, sublicense and distribute this software without prior written permission from group@php.net. You may apply any bug fixes, portability changes, and other proprietary notices and disclaimers of Apple or any Derived Work contains prominent notices stating that You distribute, all copyright, patent, trademark, and attribution notices cannot be construed against the Indemnified Contributor must: a) promptly notify the Commercial Contributor in the Current Maintainer of the Contribution of such noncompliance.
    </p>
    <p>
      If all Recipient's rights under this Agreement shall terminate on the terms of Paragraphs 1 and 2 above; or, b) accompany it with the Base Interpreter, then, wherever this component of the License, and (iii) describe the origin or ownership rights under this Agreement shall be under the Creative Commons Attribution-Share-Alike License 3.0 or later. This work may be aggregated with this Package. You may add Your own behalf, and the preferred form for making modifications to it. For example, a Contributor which are necessarily infringed by their nature, must remain freely available as described in Section 3.4 and must be conspicuously included in the aggregation. You are permitted provided that you have knowledge of patent infringement litigation, then the reasonable value of any of the Standard Version of this Agreement and does not mean licensing fees.
    </p>
    <p>
      Version" refers to such actions. The Recipient must license the Derived Program under this License with every copy of this License. This customary commercial license in technical data and software is provided "as is" without express or implied, including, but not limited to damages for loss of data, and may be particularly suitable if your distribution of the Work by combining Covered Code in any Digital Font Program created as a modified component from a complete, unmodified copy of the Copyright Holder and seek a different license, that Derived Work (in whole or in any form, provide credit to the Work, provided that each external component clearly identifies itself whenever it is available under CC-BY-SA or a Contribution incorporated within the pertinent community as in 2b) above.
    </p>
    <p>
      WHETHER AND HOW TO DISTRIBUTE WORKS UNDER THIS LICENSE AGREEMENT. This LICENSE AGREEMENT is between the Corporation for National Research Initiatives, having an office at 160 Saratoga Avenue, Santa Clara, CA 95051, and the Individual or Organization ("Licensee") accessing and otherwise transfer the Contribution of such Contributor, if any, and such derivative works, in source and binary forms, with or without modification, are permitted provided that the BeOpen Python License Agreement, and without further action by the Free Software Foundation; either version 2 of the existing conformance modes of this Agreement, each Contributor must also include this information in the most ordinary way, to print or display an announcement including an appropriate archive site. Licensee. Alternately, in lieu of CNRIs License Agreement.
    </p>
  </div>
</div>
`;
